### ISSUE TITLE
Enable trace output by setting env var

### ISSUE THREAD

**Issue by m1h43l** [sitemule/ILEvator#3]
Tracing could be done in the job log (with info or diagnostic messages) or to a stream file.

Tracing should not require a recompile of the program. It could be enabled by setting an environment variable which can be queried on service program loading (via a CPP module).

```
ILEVATOR_DEBUG_JOBLOG = 1
```
or 
```
ILEVATOR_DEBUG_STMF=/home/mihael/ilevator.trace
```

**Comment by m1h43l** [comment]
As far as I have understood it trace is "only" for debugging the TCP communication and thus differs from "normal" debug messages. We should enable this with an env var `ILEVATOR_TRACE_STMF`.

```
ADDENVVAR ILEVATOR_TRACE_STMF '/tmp/ilevator.trace'
```

**Comment by NielsLiisberg** [comment]
This is correctly understood - the ILEVATOR_TRACE_STMF envvar is the way to
go.

**Comment by m1h43l** [comment]
Implemented with commit 9c6d811faeb987d94b528a28a1e8b66243b8b942
